src/store/searches-slice.ts

```diff
@@ -48,42 +59,54 @@ const msgActionFulfilled = (
 	{ meta }: { meta: { arg: ConvActionParameters; requestId: string; requestStatus: string } }
 ): void => {
 	if (meta.arg.ids) {
-		state.conversations = state.conversations
-			? map(state.conversations, (conv) => ({
-					...conv,
-					messages: map(conv.messages, (msg) => {
-						if (includes(meta.arg.ids, msg.id)) {
-							return { ...msg, parent: FOLDERS.TRASH };
-						}
-						return msg;
-					})
-			  }))
-			: [];
-		state.messages = state.messages
-			? map(state.messages, (msg) => {
-					if (includes(meta.arg.ids, msg.id)) {
-						return { ...msg, parent: FOLDERS.TRASH };
-					}
-					return msg;
-			  })
-			: [];
```

Comment:
this code is commented intentionally.While reviewing this PR If anyone found any state update issue with msg action then let me know otherwise I will remove it because the Search is also synced with the actual Sync